inst/ms/aggreCAT.qmd

```diff
@@ -250,7 +284,11 @@ the aggregation functions. These aspects include:
     coding.
```

Comment:
L277: Maybe a note that certain ones of these need careful consideration of what minimum claim counts should be? E.g. can't calculate variance on one data point, can calculate with 2, but at what minimum does the user feel like they have a "reliable" variance estimate to use as a weight?

Reply:
Excellent point, added.